### PR TITLE
* add xcpretty argument option

### PR DESCRIPTION
**Pull request by hjanuschka**
add option to specify xcpretty args for snapshot

to get rid of my dirty `sh` call mentioned in this issue #368

### CODE REVIEW

File: lib/snapshot/options.rb [hunk 1]
```diff
@@ -105,6 +105,13 @@ def self.available_options
                                      env_name: "SNAPSHOT_CONFIGURATION",
                                      description: "The configuration to use when building the app. Defaults to 'Release'",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :xcpretty_args,
+                                     short_option: "-x",
+                                     env_name: "SNAPSHOT_XCPRETTY_ARGS",
+                                     description: "additional xcpretty arguments",
```
Comment (KrauseFx) [inline lib/snapshot/options.rb]: Could you change this to `additional`?

File: lib/snapshot/options.rb [hunk 2]
```diff
@@ -105,6 +105,13 @@ def self.available_options
...
+                                     default_value: "",
```
Comment (KrauseFx) [inline lib/snapshot/options.rb]: You shouldn't need the default value here

File: lib/snapshot/test_command_generator.rb [hunk 3]
```diff
@@ -53,7 +53,8 @@ def suffix
       end
 
       def pipe
-        ["| tee '#{xcodebuild_log_path}' | xcpretty"]
+        config = Snapshot.config
```
Comment (KrauseFx) [inline lib/snapshot/test_command_generator.rb]: Let's get rid of this line and just use `Snapshot.config` directly in the other line.

**Comment by hjanuschka** [comment]
@KrauseFx  changes made according to your feedback

**Comment by KrauseFx** [comment]
That's perfect, thank you so much @hjanuschka :+1:

**Pull request merged**